packages/react/src/components/summary-list.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ComponentProps, FunctionComponent } from "react";
+import { clx } from "../utils";
+
+const SummaryList: FunctionComponent<ComponentProps<"div">> = ({
+  children,
+  className,
+  ...props
+}) => (
+  <div className={clx("space-y-6", className)} {...props}>
+    {children}
+  </div>
+);
+
+const SummaryListHeader: FunctionComponent<ComponentProps<"div">> = ({
+  children,
+  className,
+  ...props
+}) => (
+  <h1
+    className={clx(
+      "text-txt-black-900 text-xl font-semibold leading-[30px]",
+      className,
+    )}
+    {...props}
+  >
+    {children}
+  </h1>
+);
+
+const SummaryListBody: FunctionComponent<ComponentProps<"div">> = ({
+  children,
+  className,
+  ...props
+}) => (
+  <div
+    className={clx(
+      "grid w-full grid-cols-[minmax(120px,_180px)_1fr_minmax(120px,_1fr)]",
+      "text-sm font-medium",
+      "grid-rows-auto min-h-[40px]",
+      "relative",
+      "[&>*:nth-child(n+1)]:border-otl-gray-200 [&>*:nth-child(n+1)]:border-b",
+      className,
+    )}
+    {...props}
+  >
```

Comment:
This is pretty rigid & limited to 3 cols. Need to consider if there are multiple `value` cols OR if the some `key` cols goes horizontal

Reference: https://ant.design/components/descriptions 

But, please do not follow antd API. Make the layout configurable by `SummaryListBody` props

Reply:
Okay boss, done!